### PR TITLE
Fix typo in gen_l10n_types.dart

### PR DESCRIPTION
**Pull request by userFortyTwo**
This pull request fixes a typo.

### CODE REVIEW

**Comment by andrewkolos** [comment]
LGTM; but, per the [bot message](https://github.com/flutter/flutter/pull/159035#issuecomment-2480763613), you'll have to sign the CLA before this can be reviewed and merged.

**Review by bkonyi — APPROVED** [review]
LGTM once the CLA is signed!

**Pull request merged**
